core/services/relay/evm/evm.go

```diff
@@ -619,7 +620,8 @@ func (r *Relayer) NewCCIPExecProvider(ctx context.Context, rargs commontypes.Rel
 	// to minimize misconfigure risk, make sense to wire Mantle only when Exec + Mantle + !IsSourceProvider
 	if r.chain.Config().EVM().ChainID().Uint64() == 5003 || r.chain.Config().EVM().ChainID().Uint64() == 5000 {
 		if !execPluginConfig.IsSourceProvider {
-			mantleInterceptor, iErr := mantle.NewInterceptor(ctx, r.chain.Client())
+			oracleAddress := r.chain.Config().EVM().GasEstimator().DAOracle().OracleAddress()
```

Comment:
I guess that this address is the constant @matYang was referring to in the comment you linked? Is there any way to ensure that the config has been set properly?

Reply:
It's in the defaults, so it should always be provided. The only thing I can do here is to raise an error if it's missing

Reply:
@winder do you mind 2nd look? I've added a nil check to the constructor, I think it should sufficient

Reply:
How do we check if its missing? if `oracleAddress == common.Address{}`?

Reply:
Ah nvm I see the nil check below